src/TodoList.jsx

```diff
@@ -1,35 +1,21 @@
 import PropTypes from "prop-types";
-import AddTodoForm from "./AddTodoForm";
+import TodoListItem from "./TodoListItem";
 
-const todoList = [
-  { id: 1, title: "Add new task" },
-  { id: 2, title: "Edit" },
-  { id: 3, title: "Task done" },
-];
-
-ListItem.propTypes = {
-  children: PropTypes.node.isRequired,
+TodoList.propTypes = {
+  todoList: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+      title: PropTypes.string.isRequired,
+    })
+  ).isRequired,
 };
 
-export default function TodoList() {
-  return (
-    <>
-      <AddTodoForm />
-      <UnorderedList />
-    </>
-  );
-}
-
-function UnorderedList() {
+export default function TodoList({ todoList }) {
   return (
     <ul>
       {todoList.map((task) => (
-        <ListItem key={task.id}>Title: {task.title}</ListItem>
+        <TodoListItem key={task.id} title={task.title} />
```

Comment:
`TodoListItem` must receive the entire object as prop, instead of the title alone.

From the assignment:
> Update the todo object reference to come from props

This is how:
```suggestion
        <TodoListItem key={task.id} todo={task} />
```

---

✅ You can fix it on the next assignment, which also have the same requirement.